noty-android/app/src/main/java/dev/shreyaspatil/noty/utils/ext/NavigationExt.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 Shreyas Patil
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.shreyaspatil.noty.utils.ext
+
+import androidx.fragment.app.Fragment
+import androidx.navigation.NavDirections
+import androidx.navigation.findNavController
+
+fun Fragment.navigate(
+    destination: NavDirections
+) {
+    if (isAdded) {
+        with(view?.findNavController()) {
```

Comment:
we can directly get it using `Fragment` instance as
```suggestion
        with(findNavController()) {
```